aix/generate_wazuh_packages.sh

```diff
@@ -40,12 +40,21 @@ show_help() {
   echo "    -e,  --environment            Install all the packages necessaries to build the RPM package"
   echo "    -s,  --store  <path>          Directory to store the resulting RPM package. By default: ${target_dir}"
   echo "    -p,  --install-path <path>    Installation path for the package. By default: ${install_path}"
-  echo "    -c,  --checksum <path>        Compute the SHA512 checksum of the RPM package."
+  echo "    -c,  --checksum <path>        Compute the SHA512 checksum of the RPM package. OpenSSL is required."
   echo "    -h,  --help                   Shows this help"
   echo
   exit $1
 }
 
+check_openssl() {
+  if [[ -z "$(command -v openssl)" ]] && [[ "${compute_checksums}" = "yes" ]]; then
+    echo "OpenSSL is not installed. OpenSSL is required to get the package checksum."
+    return 1
+  else
+    return 0
```

Comment:
```suggestion
    help 0
```

Reply:
It makes no sense to show the help message if there is no problem, the returns are used to terminate the execution or not depending on the following line:

```
check_openssl || exit 1
```

In case there is no problem, it would be `return 0`, and continue with the construction of the package.

As I have commented in the previous comment, it does not make sense to show the help message if OpenSSL is not installed, since we do not give the option to install it.